index.js

```diff
@@ -147,9 +147,10 @@ function getMetrics({ jobId, startTime, endTime }) {
                 coverage: hoek.reach(measures, 'coverage.history.0.value') || 'N/A',
                 tests: 'N/A'
             };
-            const total = hoek.reach(measures, 'tests.history.0.value');
-            const testErrors = hoek.reach(measures, 'test_errors.history.0.value');
-            const testFailures = hoek.reach(measures, 'test_failures.history.0.value');
+            const zero = { default: 0 };
+            const total = hoek.reach(measures, 'tests.history.0.value', zero);
```

Comment:
I think we should leave total in place. Else instead of the `N/A` which appears for cases when build doesn't use Sonarqube  it will start showing `0`

Reply:
I've changed it such that if the value does not exist, it will return a 'N/A' string. This will then result in skipping any of the test metrics computation, since I also changed the conditional to check that the value is both numeric, and !NaN